google_guest_agent/network/manager/netplan_linux.go

```diff
@@ -342,23 +345,32 @@ func (n *netplan) writeNetplanEthernetDropin(mtuMap map[string]int, interfaces,
 		logger.Debugf("Adding %s(%d) to drop-in configuration.", iface, i)
 
 		trueVal := true
+		falseVal := true
+
 		ne := netplanEthernet{
 			Match:  netplanMatch{Name: iface},
 			DHCPv4: &trueVal,
 			DHCP4Overrides: &netplanDHCPOverrides{
 				UseDomains: shouldUseDomains(i),
 			},
+			LinkLocal: []string{"ipv4"},
 		}
 
 		if mtu, found := mtuMap[iface]; found {
 			ne.MTU = &mtu
 		}
 
+		ne.DHCPv6 = &falseVal
```

Comment:
as by default dhcpv6 is disabled only, do we need this section?

Reply:
indeed. adjusted.